components/ram.c

```diff
@@ -51,4 +51,78 @@ ram_used(void)
 	       bprintf("%f", (float)(total - free - buffers - cached) / 1024 / 1024) :
 	       NULL;
 }
+#elif defined(__OpenBSD__)
+#include <sys/types.h>
+#include <sys/sysctl.h>
+
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include "../util.h"
+
+inline bool
```

Comment:
Do you need `bool`? Would `int` (using 0 and 1) not be sufficent? Saves us `<stdbool.h>`.